pkg/component/etcd/statefulset/statefulset.go

```diff
@@ -448,7 +448,7 @@ func (c *component) createOrPatch(ctx context.Context, sts *appsv1.StatefulSet,
 							Name:            "backup-restore",
 							Image:           c.values.BackupImage,
 							ImagePullPolicy: corev1.PullIfNotPresent,
-							Args:            c.values.EtcdBackupCommand,
+							Args:            c.values.EtcdBackupRestoreCommand,
```

Comment:
I think it will be better to call this `EtcdBackupRestoreArguments` since we're now passing arguments instead of a command

Reply:
Yes, that makes sense. I've made the change for `EtcdCommand` as well now. PTAL